docs/agents/pysmurf/index.rst

```diff
@@ -6,19 +6,17 @@
 Pysmurf/OCS Overview
 ====================
 
-There are three agents used to interact with `pysmurf <https://github.com/slaclab/pysmurf>`__.
+There are two agents used to interact with `pysmurf <https://github.com/slaclab/pysmurf>`__.
 The Pysmurf Controller is used to run pysmurf scripts.
 The Pysmurf Monitor listens to pysmurf messages published with the *pysmurf publisher*.
-The Pysmurf Archiver copies files pysmurf writes onto a storage node.
 There should be one controller per pysmurf instance (per smurf card),
-but one pysmurf-monitor and archiver can handle multiple instances.
+but one pysmurf-monitor can handle multiple instances.
 
-These pages will show you how to setup these three agents.
+These pages will show you how to setup these two agents.
 
 .. toctree::
     :caption: Pysmurf agents:
     :maxdepth: 3
 
     pysmurf-controller
     pysmurf-monitor
```

Comment:
I don't know how much this agent sub-section is helping. I kind of think we should just put the pysmurf-controller and monitor at the top-level in the docs. It's also kinda annoying to have to click `Pysmurf/OCS` on the sidebar before the actual agent docs.

Reply:
Agreed, I've removed it in 03bce4d.